marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.tsx

```diff
@@ -303,12 +303,12 @@ const AdvancedSettingsDialog = (props) => {
         targetDatabase: targetDatabase,
         targetFormat: targetFormat,
         permissions: targetPermissions,
-        headers: parseJSON(headers),
-        processors: parseJSON(processors),
+        headers: isEmptyString(headers) ? {} : parseJSON(headers),
```

Comment:
We have an RTL test that sets these to empty string and verifies that 'Invalid JSON' error is not thrown. Should that be changed?

Reply:
No, that test can still remain. If user chooses to set these to empty string, we convert them to valid json values.